components/list/list-item.js

```diff
@@ -219,53 +232,56 @@ class ListItem extends ListItemCheckboxMixin(ListItemRoleMixin(RtlMixin(LitEleme
 	render() {
 		const classes = {
 			'd2l-visible-on-ancestor-target': true,
-			'focusing': this._focusing,
+			'd2l-focusing': this._focusing,
+			'd2l-hovering': this._hovering,
 		};
 		const contentClasses = {
 			'd2l-list-item-content': true,
 			'd2l-list-item-content-extend-separators': this._extendSeparators,
-			'hovering': this._hoveringLink,
-			'focusing': this._focusingLink,
+			'd2l-hovering': this._hoveringLink,
+			'd2l-focusing': this._focusingLink,
 		};
 
 		return html`
-			<d2l-list-item-generic-layout
-				@focusin="${this._handleFocusIn}"
-				@focusout="${this._handleFocusOut}"
-				class="${classMap(classes)}"
-				data-breakpoint="${this._breakpoint}"
-				data-separators="${ifDefined(this._separators)}"
-				?grid-active="${this.role === 'rowgroup'}">
-				${ this.draggable ? html`
-				<div slot="outside-control"></div>
-				` : nothing }
-				${this.selectable ? html`
-				<div slot="control">
-					${ this._renderCheckbox() }
-				</div>
-				<div slot="control-action">${ this._renderCheckboxAction('', this._contentId) }</div>
-				` : nothing }
-				${ this.href ? html`
-				<a slot="content-action"
-					href="${this.href}"
-					aria-labelledby="${this._contentId}"
-					@mouseenter="${this._handleMouseEnterLink}"
-					@mouseleave="${this._handleMouseLeaveLink}"
-					@focus="${this._handleFocusLink}"
-					@blur="${this._handleBlurLink}"></a>
-				` : nothing }
-				<div slot="content"
-					class="${classMap(contentClasses)}"
-					id="${this._contentId}">
-					<slot name="illustration"></slot>
-					<slot></slot>
-				</div>
+			${this._renderTopPlacementMarker(html`<d2l-list-item-placement-marker></d2l-list-item-placement-marker>`)}
+			${this._renderDropTarget()}
+			<div class="d2l-list-item-drag-image">
+				<d2l-list-item-generic-layout
+					@focusin="${this._onFocusIn}"
+					@focusout="${this._onFocusOut}"
+					class="${classMap(classes)}"
+					data-breakpoint="${this._breakpoint}"
+					data-separators="${ifDefined(this._separators)}"
+					?grid-active="${this.role === 'rowgroup'}">
+					${this._renderDragHandle(this._renderOutsideControl)}
+					${this._renderDraggableArea(this._renderOutsideControlAction)}
+					${this.selectable ? html`
+					<div slot="control">${ this._renderCheckbox() }</div>
+					<div slot="control-action">${ this._renderCheckboxAction('', this._contentId) }</div>
+					` : nothing }
```

Comment:
In another PR this would be great to clean up and make consistent with `listItemDragDropMixin`